website/content/docs/what-is-boundary.mdx

```diff
@@ -36,26 +46,58 @@ and self-managed infrastructures without exposing your network or the use of
+[grants](/docs/concepts/security/permissions), which provide the ability to
+perform actions within the system.
 
-Having different access workflows for various infrastructure and identity providers doesn’t scale well for administrators or users. Multi-cloud organizations have a growing need to standardize access through a single workflow. Boundary creates a centralized layer of identity-based authentication and authorization to manage access to the infrastructure, regardless of the platform on which it resides.
+### Resource Management
 
-### Single Sign-on with Integrated Secrets Management
+Boundary enables flexible management of the hosts and services for which it can
+broker access. Boundary administrators define [host
+catalogs](/docs/concepts/domain-model/host-catalogs) that contain information
+about [hosts](/docs/concepts/domain-model/hosts). These hosts are then collected
+into [host sets](/docs/concepts/domain-model/host-sets) which represent sets of
+equivalent hosts. Finally, [targets](/docs/concepts/domain-model/targets) tie
+together host sets with connection information. Final access to a resource is
+granted via [roles](/docs/concepts/domain-model/roles) that provide
+authorization to create sessions against these targets.
 
-Boundary enables a single sign-on access model with authentication from trusted identity providers, such as Azure Active Directory, Auth0, and Okta. Once authenticated, users may create sessions with integrated credential management from HashiCorp Vault without the need to re-authenticate.
+### Filtering
 
-### Session Monitoring
+Some parts of Boundary support filters for various purposes. For a description
+of the filter syntax, see the [filtering](/docs/concepts/filtering) page. See
+the docs pages for the individual resources or capabilities where filters are
+supported for the specific inputs and examples with those inputs.
 
-Boundary provides session monitoring capabilities that give security administrators visibility into user access. Sessions are logged and consumable via the Boundary administrator UI as well as business intelligence (BI) and security information and event management (SIEM) tools.
+-> **Tip**: Learn more about Boundary [use cases](/docs/use-cases).
+
+## What is HCP Boundary?
+
+Boundary offers two types of deployment options. The first option is the *self-managed* approach, which we have discussed on this page. The self-managed approach enables organizations to proxy all session data through their own network while still providing the convenience of a managed service. **HCP Boundary**, on the other hand,  is a *fully-managed*, cloud-based workflow that enables secure connections to remote hosts and critical systems across cloud and on-premise environments. In the fully-managed deployment model, both the control plane and worker nodes are managed by HashiCorp. Refer to the [HCP Boundary](https://cloud.hashicorp.com/docs/boundary
+) documentation to learn more.
```

Comment:
I'd like a second opinion on the language in this section, since there are really three different deployment models being described here?

Self-managed: there is Boundary OSS, which is truly "self-managed".
HCP with self-managed workers: HCP Boundary hosts the control plane, while the organization self-manages their own workers.
Fully-managed: HCP hosts both the control plane and worker nodes.

On the Concepts page, we state that:

> There are several options for deploying Boundary:
> 
> 1. HCP Boundary as a fully-managed service
> 2. Boundary OSS for self-managed deployments
> 3. Dev mode for testing and learning purposes

We should make these statements consistent, and probably distinguish that HCP Boundary supports fully-managed and self-proxied deployment options, so folks don't assume Boundary OSS is the only option there.

@PPacent and @xingluw might want to weigh in on this bit.

Reply:
I would prefer it be stated in 2 options: 
- OSS self managed
- HCP managed solution
Where the managed solution has an option of private workers
The dev mode mentioned above would be part of the OSS self managed

Reply:
@jefferai mentioned to me that we shouldn't call dev mode a "deployment" option, either... but I see that's what's stated under the [concepts overview page](https://www.boundaryproject.io/docs/concepts) too

Reply:
I tweaked the content a bit based on xingluw's feedback.